### DISCUSSION TITLE
"Tracked objects" Labels order in Explore window

### DISCUSSION THREAD

**Discussion by 18923** [blakeblackshear/frigate#15163] [Ask A Question] [answered]
What determines the Label sort order in the Explore window? Can the label (Car, Person, Dog) order be customized?

81Xats

**Comment by hawkeye217** [comment]
If you're referring to the summary view (displaying 10 latest tracked objects of each label), it's arranged by the number of tracked objects per label in descending order.

It cannot be customized.

**Reply by 18923** [reply]
Looks like my example image did not make it through the firs time, you should see it now.

**Comment by 18923** [comment]
![frigate](https://github.com/user-attachments/assets/d6afc152-3520-407f-9e26-bb2b4490bb52)

**Reply by hawkeye217** [reply, marked as answer]
Yes, this is the summary view in Explore.
